### PR TITLE
Add RelayAddr/Chan to Alarmdecoder

### PR DESCRIPTION
**Pull request by digiblur**
**Description:**

Adds the two optional configuration options referenced in the PR to the component.
https://github.com/home-assistant/home-assistant/pull/15242

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#15242

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/alarmdecoder.markdown [hunk 1]
```diff
@@ -43,6 +43,11 @@ alarmdecoder:
     02:
       name: 'Front Door'
       type: 'opening'
+    03:
```
Comment (frenck) [inline source/_components/alarmdecoder.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

File: source/_components/alarmdecoder.markdown [hunk 2]
```diff
@@ -55,3 +60,5 @@ Configuration variables:
...
 - **rfid** (*Optional*): The RF serial-number associated with RF zones.  Providing this field allows Home Assistant to associate raw sensor data to a given zone, allowing direct monitoring of the state, battery, and supervision status.
+- **relayaddr** (*Optional*): Address of the relay expander board to associate with the zone. (ex: 12, 13, 14, or 15). Typically used in cases where a panel will not send bypassed zones such as motion during an armed home state, the Vista 20P is an example of this.  Alarmdecoder can emulate a zone expander board and the panel can be programmed to push zone events to this virtual expander. This allows the bypassed zone binary sensors to be utilized. One example is using bypassed motion sensors at night for motion based automated lights while the system is armed with the motion sensor bypassed.
```
Comment (frenck) [inline source/_components/alarmdecoder.markdown]: `motion based` -> `motion-based`

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
Thank you, @digiblur for this PR.
I've left 2 comments (of 1 is a typo). Could you take a look? 👍

File: source/_components/alarmdecoder.markdown (hunk 1 above)
Comment (digiblur) [reply source/_components/alarmdecoder.markdown]: I do agree with the changes.  To follow with this the rule rfid should also be removed from zone 1 that is not part of my current proposed changes.

File: source/_components/alarmdecoder.markdown (hunk 2 above)
Comment (digiblur) [reply source/_components/alarmdecoder.markdown]: All sounds good, new to the whole PR process/procedures with HA, how do you accept these changes?  Do I make them on my side and commit them?

**Comment by digiblur** [comment]
Corrections done.

**Review by frenck — DISMISSED** [review]

**Comment by frenck** [comment]
Thanks for the quick actions @digiblur!

✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by digiblur** [comment]
Thanks for the review and help with my first PR of this project @frenck !

File: source/_components/alarmdecoder.markdown [hunk 3]
```diff
@@ -55,3 +55,5 @@ Configuration variables:
...
 - **rfid** (*Optional*): The RF serial-number associated with RF zones.  Providing this field allows Home Assistant to associate raw sensor data to a given zone, allowing direct monitoring of the state, battery, and supervision status.
+- **relayaddr** (*Optional*): Address of the relay expander board to associate with the zone. (ex: 12, 13, 14, or 15). Typically used in cases where a panel will not send bypassed zones such as motion during an armed home state, the Vista 20P is an example of this.  Alarmdecoder can emulate a zone expander board and the panel can be programmed to push zone events to this virtual expander. This allows the bypassed zone binary sensors to be utilized. One example is using bypassed motion sensors at night for motion-based automated lights while the system is armed with the motion sensor bypassed.
```
Comment (MartinHjelmare) [inline source/_components/alarmdecoder.markdown]: Looks like there are two spaces between "this." and "Alarmdecoder can emulate".

**Comment by digiblur** [comment]
Fixed the line above it too as I noticed the double space there as well.

**Review by MartinHjelmare — APPROVED** [review]

**Pull request merged**